### ISSUE TITLE
Issue with pull policy for core services

### ISSUE THREAD

**Issue by elebeaup** [openfaas/faas-netes#322]
I'm trying to deploy on Kubernetes in offline (Local cluster/Docker for Desktop)

## Expected Behaviour
Openfaas is deployed without errors.

## Current Behaviour
The gateway fail to deploy. I have the following error :
```Failed to pull image "openfaas/openfaas-operator:0.8.9": rpc error: code = Unknown desc = Error response from daemon: Get https://registry-1.docker.io/v2/: net/http: request canceled while waiting for connection```

## Possible Solution
For the gateway component, the imagePullPolicy is set to **Always** instead of {{ .Values.openfaasImagePullPolicy }}

## Steps to Reproduce (for bugs)
1. Deploy OpenFaaS from the helm chart repo:
```
helm upgrade openfaas chart/openfaas --install \
  --set "faasnetesd.imagePullPolicy=IfNotPresent" \
  --set "openfaasImagePullPolicy=IfNotPresent" \
  --namespace openfaas  \
  --set functionNamespace=openfaas-fn \
  --set operator.create=true
```

## Context
- Offline
- Local cluster

## Your Environment
* Docker version `docker version` (e.g. Docker 17.0.05 ):
Docker 18.03.0-ce

* What version and distriubtion of Kubernetes are you using? `kubectl version`
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.3", GitCommit:"2bba0127d85d5a46ab4b778548be28623b32d0b0", GitTreeState:"clean", BuildDate:"2018-05-21T09:17:39Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.3", GitCommit:"2bba0127d85d5a46ab4b778548be28623b32d0b0", GitTreeState:"clean", BuildDate:"2018-05-21T09:05:37Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}

* Operating System and version (e.g. Linux, Windows, MacOS):
Windows 10

**Comment by alexellis** [comment]
@stefanprodan PTAL?

**Comment by dedene** [comment]
@alexellis just curious, have you experienced this issue while testing: https://github.com/openfaas-incubator/openfaas-operator/issues/56 ?

**Comment by alexellis** [comment]
Hi @dedene sorry I missed this follow-up comment.

It looks like @stefanprodan fixed the issue you had in #6? Can you comment please?

Alex
